packages/cli/src/WorkflowExecuteAdditionalData.ts

```diff
@@ -646,13 +646,18 @@ function hookFunctionsSaveWorker(): IWorkflowExecuteHooks {
 			async function (this: WorkflowHooks, runData: IRun): Promise<void> {
 				const { executionId, workflowData: workflow } = this;
 
-				void internalHooks.onWorkflowPostExecute(executionId, workflow, runData);
 				eventService.emit('workflow-post-execute', {
-					workflowId: workflow.id,
-					workflowName: workflow.name,
-					executionId,
-					success: runData.status === 'success',
-					isManual: runData.mode === 'manual',
+					audit: {
```

Comment:
why not emit two separate events?

Reply:
Other option would be to keep the event payload generic enough so both `telemetry-event-relay` and `audit-event-relay` can then derive whatever data they need from that. Tho that makes it then implicit what happens based on the sent event. Then again that's usually the point of using event buses: to decouple consumers from producers. Now the coupling is there, as we explicitly define the data separately for each consumer.